app/views/invite_mailer/invite.html.erb

```diff
@@ -0,0 +1,3 @@
+Hello <%= @name %>,
+
+<%= @sender_name %> has invited you to join Brownfield of Dreams. You can create an account <%= link_to 'here', signup_url %>.
```

Comment:
`@sender_name` verified in pry. it works!

Reply:
Sounds good